src/__tests__/__snapshots__/typescript.js.snap

```diff
@@ -1,7 +1,10 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Typescript: Typescript expected failures 1`] = `
-"test/should-fail.test.tsx(10,3): error TS2345: Argument of type '{ fillRule: \\"cat\\"; }' is not assignable to parameter of type 'StyleArgument<SVGProperties, {}>'.
+"test/glamorous.test.tsx(537,43): error TS2322: Type '{}' is not assignable to type 'IntrinsicAttributes & CSSProperties & ExtraGlamorousProps & HTMLProps<HTMLDivElement> & { childre...'.
```

Comment:
Other snapshot updates are acceptable, but this one should not be included.
You should update `glamorous.test.tsx` instead of update snapshot.
However, https://github.com/paypal/glamorous/pull/372 will solve this error without changing `glamorous.test.tsx` or snapshot.